packages/stentor-constants/CHANGELOG.md

```diff
@@ -0,0 +1,56 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [1.14.1-alpha.6](https://github.com/stentorium/stentor/compare/stentor-constants@1.14.1-alpha.5...stentor-constants@1.14.1-alpha.6) (2020-01-03)
```

Comment:
Do we care about any of this for an initial public release?

Reply:
Not really but they exist https://www.npmjs.com/package/stentor-models so we should keep the record of them